SwiftExample/Podfile

```diff
@@ -5,12 +5,10 @@
 
 target 'SwiftExample' do
 
-pod 'JSQMessagesViewController'
-
+pod 'JSQMessagesViewController', :git => 'https://github.com/jessesquires/JSQMessagesViewController.git', :branch => 'develop'
```

Comment:
oh, i didn't realize this before.

we should be point the spec to `../JSQMessagesViewController.podspec` locally 

(same below)